### ISSUE TITLE
Latest Version issue

### ISSUE THREAD

**Issue by Sawfat** [wmtorode/MW5-SaveEditor#36]
i am using latest version with modded MW 5 using YAML and other related mods,everytime i load the edited save it become like corrupted,all my lance gone but the box of which lance appeare like the image and stat are still there,all my mech gone,starmap also bugged,all conflict zone gone,all market items gone.

**Comment by Sawfat** [comment]
Update: version in nexus are still usable without error,so if anyone encounter any problem,try that one in nexus

**Comment by Kstims** [comment]
I was able to use version 1.6.3, however it was unable to scan my mods. Running YAML and its derivatives like yet another mech lab, I would scan my mods folder to create the cache, but when I would attempt to add a modded weapon, engine, part, etc. it wouldn't show up in the list unless I already had one instance of said item in my inventory.

Also, I do not know if it is the YAML mods I was running or editing the save, but eventually my friends were completely unable to join my game, they would be stuck in the loading screens forever.

I tried to create a new career with the exact same mods and they were able to load into the game just fine on a fresh install. Again dont know if that is related to the save editing causing corruptions they weren't able to load, the mods having issues, or something else.
